packages/ibm-products/src/components/Checklist/Checklist.js

```diff
@@ -0,0 +1,336 @@
+              </Button>
+            )}
+          </header>
+        )}
+
+        <div id={listContainerId} className={`${blockClass}__content-outer`}>
+          <div className={`${blockClass}__content-inner`}>
+            <section className={cx(`${blockClass}__body`)}>
+              {taskLists.map((list, index) => {
+                return (
+                  <div
+                    className={`${blockClass}__list-group`}
+                    key={`${list.title}-${index}`}
+                  >
+                    {list.title && (
+                      <h3
+                        title={list.title}
+                        className={`${blockClass}__list-title`}
+                      >
+                        {list.title}
+                      </h3>
+                    )}
+                    <ol className={`${blockClass}__list`}>
+                      {list.tasks.map((item, index) => {
+                        return (
+                          <li
+                            className={`${blockClass}__list-item`}
+                            key={`${item.label}-${index}`}
+                          >
+                            <ChecklistIcon kind={item.kind} />
+
+                            {typeof item.onClick === 'function' ? (
+                              <Button
+                                className={cx(`${blockClass}__button`, {
+                                  [`${blockClass}__button--error`]:
+                                    item.kind === 'error',
+                                })}
+                                onClick={() => {
+                                  item.onClick(item);
+                                }}
+                                size="sm"
+                              >
+                                <div title={item.label}>{item.label}</div>
+                              </Button>
+                            ) : (
+                              <span
+                                className={cx(
+                                  `${blockClass}__label`,
+                                  `${blockClass}__label--${item.kind}`
+                                )}
+                                title={item.label}
+                              >
+                                {item.label}
+                              </span>
+                            )}
+                          </li>
+                        );
+                      })}
+                    </ol>
+                  </div>
+                );
+              })}
+            </section>
+
+            {viewAllLabel && (
+              <footer className={`${blockClass}__footer`}>
+                <Button
+                  className={cx(
+                    `${blockClass}__button`,
+                    `${blockClass}__view-all`
+                  )}
+                  onClick={handleClickViewAll}
+                  size="sm"
+                >
+                  <div>{viewAllLabel}</div>
```

Comment:
Don't think you need the extra `div` here either.

Reply:
We do. Please see https://github.com/carbon-design-system/ibm-products/pull/3626#discussion_r1365597488 above.